src/components/navigation-tabs/BackToInstructor.jsx

```diff
@@ -1,16 +1,19 @@
 import React from 'react';
+import { useParams } from 'react-router-dom';
 
 import { getConfig } from '@edx/frontend-platform';
 import { FormattedMessage } from '@edx/frontend-platform/i18n';
 import { Button, Icon } from '@edx/paragon';
 import { ArrowBack } from '@edx/paragon/icons';
 
 export default function BackToInstructor() {
+  const { courseId } = useParams();
```

Comment:
This should come in as a prop, so that `useParams()` is used in as few places as possible.

Reply:
Yeah, I don't have strong opinon on either, I though useParams, is simlair to global context. Is it better for optimization

Reply:
In this case, it means we'll have a single place in the code that defines courseId.  It makes it easier to debug and test.

Plus, this is what props are for.  :shrug: